test/functions/hasPowerShell3.spec.ts

```diff
@@ -1,15 +1,43 @@
 import test from 'ava'
 
 import hasPowerShell3 from '@/src/functions/hasPowerShell3'
+import mockDependencies from '@/test/__helpers__/mockDependencies'
 
 
 test('windows: release <=6 must NOT have PowerShell 3', t => {
-	t.is(hasPowerShell3('5.0.12'), false)
-	t.is(hasPowerShell3('6.1.0'), false)
+	const dependencies = mockDependencies({
+		platform: 'win32',
+	})
+
+	dependencies.release = '5.0.12'
+	t.is(hasPowerShell3(dependencies), false)
```

Comment:
I prefer to not edit the `dependencies` object by using destructuration like this:

```ts
t.is(hasPowerShell3({
	...dependencies,
	release: '5.0.12',
}), false)
```